accounts/lists.py

```diff
@@ -0,0 +1,27 @@
+# User types list passed into dropdown of same name for user selection used
+# in models.py & forms.py
+user_types = [
+    ('', 'Select Post Category'),
+    ('participant', 'Participant'),
+    ('staff', 'Staff'),
+    ('admin', 'Admin'),
+]
+
+# LMS Modules list passed into dropdown of same name for user selection used
+# in modules.py & forms.py
+lms_modules = [
+    ('', 'Select Learning Stage'),
+    ('programme preliminaries', 'Programme Preliminaries'),
```

Comment:
If you are tying to make this more machine readable I would suggest to also replace spaces with underscores.

Reply:
Machine Readable for the backend values only? So for example, refactor to:
```python
LMS_MODULES_CHOICES = [
    ('programme_preliminaries', 'Programme Preliminaries')
]
```